packages/react-workspace/nimble-react/styles/fonts.scss

```
@@ -1,4 +1,4 @@
 // Nimble React Fonts SCSS
-// Only @import once in the global styles.scss
+// Only @forward/@use once in the global styles.scss
```

Based on above comments
```suggestion
// Only @use once in the global styles.scss
```